qemu/tests/watchdog.py

```diff
@@ -283,12 +283,14 @@ def hotplug_unplug_watchdog_device():
     # main procedure
     test_type = params.get("test_type")
     error.context("'%s' test starting ... " % test_type, logging.info)
-    error.context("Boot VM with WDT(Device:'%s', Action:'%s'),and try to login"
-                  % (watchdog_device_type, watchdog_action), logging.info)
-    params["start_vm"] = "yes"
-    env_process.preprocess_vm(test, params, env, params.get("main_vm"))
-    vm = env.get_vm(params["main_vm"])
-    session = vm.wait_for_login(timeout=timeout)
+
+    if test_type != "check_watchdog_support":
+        error.context("Boot VM with WDT(Device:'%s', Action:'%s'), and login"
+                      % (watchdog_device_type, watchdog_action), logging.info)
+        params["start_vm"] = "yes"
+        env_process.preprocess_vm(test, params, env, params.get("main_vm"))
+        vm = env.get_vm(params["main_vm"])
+        session = vm.wait_for_login(timeout=timeout)
 
     if params.get("setup_runlevel") == "yes":
```

Comment:
I think this if statement section should be included into the fix as well. Others look good to me.